diff --git a/boards/heltec_v4.json b/boards/heltec_v4.json
index 9827be83f..961783f80 100644
--- a/boards/heltec_v4.json
+++ b/boards/heltec_v4.json
@@ -2,7 +2,7 @@
   "build": {
     "arduino": {
       "ldscript": "esp32s3_out.ld",
-      "partitions": "default_16MB.csv",
+      "partitions": "partitions.csv",
       "memory_type": "qio_qspi"
     },
     "core": "esp32",
diff --git a/variants/esp32s3/heltec_v4/platformio.ini b/variants/esp32s3/heltec_v4/platformio.ini
index 72c53ded0..09b643d54 100644
--- a/variants/esp32s3/heltec_v4/platformio.ini
+++ b/variants/esp32s3/heltec_v4/platformio.ini
@@ -2,7 +2,7 @@
 extends = esp32s3_base
 board = heltec_v4
 board_check = true
-board_build.partitions = default_16MB.csv
+board_build.partitions = partitions.csv
 build_flags = 
   ${esp32s3_base.build_flags}
   -D HELTEC_V4
