diff --git a/variants/esp32s3/heltec_v3/platformio.ini b/variants/esp32s3/heltec_v3/platformio.ini
index a8e9f5e..9d53f01 100644
--- a/variants/esp32s3/heltec_v3/platformio.ini
+++ b/variants/esp32s3/heltec_v3/platformio.ini
@@ -13,6 +13,6 @@
 board = heltec_wifi_lora_32_V3
 board_level = pr
 board_check = true
-board_build.partitions = default_8MB.csv
+board_build.partitions = partitions.csv
 build_flags = 
   ${esp32s3_base.build_flags}
